contracts/modules/cover/QuotationData.sol

```diff
@@ -66,7 +67,7 @@ contract QuotationData is LegacyMasterAware {
 
   event CoverStatusEvent(uint indexed cid, uint8 statusNum);
 
-  constructor(address _authQuoteAdd, address _kycAuthAdd) public {
+  constructor(address _authQuoteAdd, address _kycAuthAdd, address masterAddress) public {
```

Comment:
what's the goal of this masterAddress parameter?

We're not updating this contract